src/color.js

```diff
@@ -168,12 +164,6 @@ export default class Color {
 		if (exports.register) {
 			exports.register(Color);
 		}
-		else if (exports.default) {
-			Color.defineFunction(exports.default.name, exports.default);
-		}
-		else if (typeof exports === "function") {
-			Color.defineFunction(exports);
-		}
```

Comment:
Remove the corresponding type definitions: https://github.com/LeaVerou/color.js/blob/main/types/src/color.d.ts#L105 and https://github.com/LeaVerou/color.js/blob/main/types/src/color.d.ts#L107

Reply:
Done!